### PR TITLE
Initial_recipe

### PR DESCRIPTION
**Pull request by ritumd**
## Checklist before submitting

- [X] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by alexlang74 — APPROVED** [review]

**Comment by cdeepali** [comment]
@ritumd, I think we can clean the [win] specific code from the recipe.

File: LICENSE.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 BSD 3-clause license
-Copyright (c) 2015-2021, conda-forge contributors
+Copyright (c) 2022, IBM
+Copyright (c) 2015-2022, conda-forge contributors
```
Comment (cdeepali) [inline LICENSE.txt]:
```suggestion
Copyright (c) 2015-2021, conda-forge contributors
```

**Review by cdeepali — APPROVED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,49 +12,42 @@ source:
...
   - name: libmamba
     version: {{ libmamba_version }}
     script: build_mamba.sh                       # [unix]
-    script: build_mamba.bat                      # [win]
     build:
       run_exports:
         - {{ pin_subpackage('libmamba', max_pin='x.x') }}
       ignore_run_exports:
-        - spdlog
+        - spdlog {{spdlog}}
```
Comment (cdeepali) [inline recipe/meta.yaml]:
```suggestion
        - spdlog
```

File: recipe/meta.yaml [hunk 3]
```diff
@@ -63,33 +56,32 @@ outputs:
...
     version: {{ libmambapy_version }}
     script: build_mamba.sh                       # [unix]
-    script: build_mamba.bat                      # [win]
     build:
       string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
       run_exports:
         - {{ pin_subpackage('libmambapy', max_pin='x.x') }}
       ignore_run_exports:
-        - spdlog
+        - spdlog {{spdlog}}
```
Comment (cdeepali) [inline recipe/meta.yaml:64]:
```suggestion
        - spdlog
```

**Pull request merged**